addons/LPCAnimatedSprite/LPCAnimatedSprite2D.gd

```diff
@@ -94,9 +101,13 @@ func _ready():
 		LoadAnimations()
 		
 func play(animation: LPCAnimation):
-	var sprites = get_children()
+	var sprites = get_children() as Array[AnimatedSprite2D]
 	for sprite in sprites:
-		sprite.play(AnimationData[animation].Name)
+		if sprite.sprite_frames.has_animation(AnimationData[animation].Name):
+			sprite.visible = true
+			sprite.play(AnimationData[animation].Name)
+		else:
+			sprite.visible = false
```

Comment:
on the visible? I did that only to avoid warnings when calling play(WALK_LEFT), because the oversized do not have that animation. So the Godot will be flooded with warnings. But whithout another animation to replace whe frames kept on the screen

Reply:
Good point!